src/pages/events/eventsearch-pipe.ts

```diff
@@ -0,0 +1,75 @@
+import {Pipe, Component, PipeTransform} from '@angular/core';
```

Comment:
If this is another popover now and not a pipe, rename it, put it with the other popovers, and take out the unused imports